Build failure in beam-core-0.9.2.0 with GHC-8.2.2

```
Database/Beam/Backend/SQL/Builder.hs:58:10: error:
    Not in scope: type constructor or class ‘Semigroup’
   |
58 | instance Semigroup SqlSyntaxBuilder where
   |          ^^^^^^^^^
```

--- a/beam-core/beam-core.cabal
+++ b/beam-core/beam-core.cabal
@@ -60,7 +60,7 @@ library
 
                        Database.Beam.Schema.Lenses
 
-  build-depends:       base         >=4.9     && <5.0,
+  build-depends:       base         >=4.11    && <5.0,
                        aeson        >=0.11    && <2.1,
                        text         >=1.2.2.0 && <1.3,
                        bytestring   >=0.10    && <0.12,

